src/components/Link.js

```diff
@@ -85,8 +85,9 @@ class CSVLink extends React.Component {
         return this.props.asyncOnClick
           ? this.handleAsyncClick(event)
           : this.handleSyncClick(event);
+      } else {
```

Comment:
why would this change anything? it doesn't get called since the previous line has a return?